<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EAWaveletsForm</class>
 <widget class="QWidget" name="EAWaveletsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>199</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ptInput" name="Master"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="ptInput" name="Level1"/>
   </item>
   <item>
    <widget class="ptInput" name="Level2"/>
   </item>
   <item>
    <widget class="ptInput" name="Level3"/>
   </item>
   <item>
    <widget class="ptInput" name="Level4"/>
   </item>
   <item>
    <widget class="ptInput" name="Level5"/>
   </item>
   <item>
    <widget class="ptInput" name="Level6"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ptInput</class>
   <extends>QProgressBar</extends>
   <header>ptInput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
